lib/src/builder/codegen/util.dart

```diff
@@ -66,3 +67,22 @@ String generatedMixinWarningCommentLine(TypedMapNames mixinNames, {@required boo
 
   return value;
 }
+
+void generatePropsMeta(
+  StringBuffer buffer,
+  List<Identifier> mixins, {
+  String classType = 'PropsMetaCollection',
+  String fieldName = 'propsMeta',
+  bool isConst = true,
+}) {
+  buffer
+    ..writeln()
+    ..writeln('  @override')
+    ..writeln('  $classType get $fieldName => ${isConst ? 'const' : ''} $classType({');
```

Comment:
Why is the instance conditionally const? Can't it always be const?

Reply:
I might be missing something, but not with the current implementation of `InstancePropsMeta` because it has instance fields. If I change that to not keep that state though, then that will clean this up as well!

Reply:
Updated this in ea0e4aa!